IBAnimatable/GradientDesignable.swift

```diff
@@ -354,3 +356,23 @@ public extension GradientDesignable where Self: UIView {
     }
   }
 }
+
+private class GradientView: UIView {
+
+  // MARK: - Life cycle
+  
+  init(frame: CGRect, layer: CAGradientLayer) {
```

Comment:
I like the idea that the protocol extension doesn't have the knowledge how `GradientView` works. I mean, it has to know how create a gradient and customise the layer... but the view part is not really related to the protocol. That's the only reason I created a custom init, but it could be also a `configGradientView()`, that may conform more the rest of the project.

What do you think?